app/src/main/java/com/nextcloud/talk/activities/CallActivity.kt

```diff
@@ -381,6 +385,9 @@ class CallActivity : CallBaseActivity() {
         canPublishAudioStream = extras.getBoolean(KEY_PARTICIPANT_PERMISSION_CAN_PUBLISH_AUDIO)
         canPublishVideoStream = extras.getBoolean(KEY_PARTICIPANT_PERMISSION_CAN_PUBLISH_VIDEO)
         isModerator = extras.getBoolean(KEY_IS_MODERATOR, false)
+        isOneToOneConversation = extras.getBoolean(KEY_IS_ROOM_ONE_TO_ONE, false)
```

Comment:
i think it's enough to just check for the one-to-one key.
So no need to introduce KEY_IS_ROOM_GROUP. If it's not a one-to-one conversation do the handling for the click listeners as it's a group conversation.